.releaserc

```diff
@@ -1,26 +1,19 @@
 {
+  "debug": true,
```

Comment:
I like to always enable debug with semantic-release because it doesn't add an order of magnitude to the volume of output and it's handy to have some of that extra info already displayed in the build logs if something goes wrong :)